core/src/main/java/bisq/core/trade/statistics/TradeStatistics3.java

```diff
@@ -0,0 +1,355 @@
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.core.trade.statistics;
+
+import bisq.core.monetary.Altcoin;
+import bisq.core.monetary.AltcoinExchangeRate;
+import bisq.core.monetary.Price;
+import bisq.core.monetary.Volume;
+import bisq.core.offer.OfferUtil;
+
+import bisq.network.p2p.storage.payload.CapabilityRequiringPayload;
+import bisq.network.p2p.storage.payload.PersistableNetworkPayload;
+import bisq.network.p2p.storage.payload.ProcessOncePersistableNetworkPayload;
+
+import bisq.common.app.Capabilities;
+import bisq.common.app.Capability;
+import bisq.common.crypto.Hash;
+import bisq.common.proto.ProtoUtil;
+import bisq.common.util.CollectionUtils;
+import bisq.common.util.ExtraDataMapValidator;
+import bisq.common.util.JsonExclude;
+import bisq.common.util.Utilities;
+
+import com.google.protobuf.ByteString;
+
+import org.bitcoinj.core.Coin;
+import org.bitcoinj.utils.ExchangeRate;
+import org.bitcoinj.utils.Fiat;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Charsets;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Map;
+import java.util.Optional;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * This new trade statistics class uses only the bare minimum of data.
+ * Data size is about 50 bytes in average
+ */
+@Slf4j
+@Getter
+public final class TradeStatistics3 implements ProcessOncePersistableNetworkPayload, PersistableNetworkPayload,
+        CapabilityRequiringPayload {
+
+    // This enum must not change the order as we use the ordinal for storage to reduce data size.
+    // The payment method string can be quite long and would consume 15% more space.
+    // When we get a new payment method we can add it to the enum at the end. Old users would add it as string if not
+    // recognized.
+    private enum PaymentMethodMapper {
+        OK_PAY,
+        CASH_APP,
+        VENMO,
+        AUSTRALIA_PAYID, // seems there is a dev trade
+        UPHOLD,
+        MONEY_BEAM,
+        POPMONEY,
+        REVOLUT,
+        PERFECT_MONEY,
+        SEPA,
+        SEPA_INSTANT,
+        FASTER_PAYMENTS,
+        NATIONAL_BANK,
+        JAPAN_BANK,
+        SAME_BANK,
+        SPECIFIC_BANKS,
+        SWISH,
+        ALI_PAY,
+        WECHAT_PAY,
+        CLEAR_X_CHANGE,
+        CHASE_QUICK_PAY,
+        INTERAC_E_TRANSFER,
+        US_POSTAL_MONEY_ORDER,
+        CASH_DEPOSIT,
+        MONEY_GRAM,
+        WESTERN_UNION,
+        HAL_CASH,
+        F2F,
+        BLOCK_CHAINS,
+        PROMPT_PAY,
+        ADVANCED_CASH,
+        BLOCK_CHAINS_INSTANT
+    }
+
+    private final String currency;
+    private final long price;
+    private final long amount;
+    private final String paymentMethod;
```

Comment:
Because its more flexible for future changes. If we add a new method and a not updated user gets that data it will now show the string even the payment method is not included yet, otherwise it would show Unknown.

Reply:
As we store only the ordinal as string, its from data size same...only when paymentMethod is stored as full string we get a bit more data.